spec/models/chargeback_container_project_new_chargeback_spec.rb

```diff
@@ -1,2 +1,2 @@
-Settings[:new_chargeback] = '1'
+ChargebackContainerImage.include(Chargeback::New)
```

Comment:
The fact that any of this is being done at this context is the problem, especially since our tests run in random order.